### DISCUSSION TITLE
Create a job board website

### DISCUSSION THREAD

**Discussion by johnhuichen** [ReaVNaiL/New-Grad-2024#529] [Ideas]
Hi everyone, 

I am wondering if anyone think it will be useful if I create a job board website specifically for new grads?

I am debating the value of it because it does take some time to make a good website, and there are so many job boards out there. However this is an idea I have been toying with for the past 4 weeks.

Let me know your opinion, and let me know the biggest pain point for you in job searching. There must be something we can do to ease the process of landing a first job

Quick update:

It's really amazing to see so many people wanting to help! It's a big encouragement, and thank you for that!

The technical part of building a website is not a problem at all. I would know exactly how long it's going to take to build the first mvp given all the feature requirements. However at the moment I am checking if this website would be useful once it's built. Nothing is worse than going live after months of work and the web traffic is flat.

I noticed that this repository links to a discord channel owned by cscareers.dev. As far as I can tell they already have an amazing website dedicated to the same cause (https://www.cscareers.dev/) so please check them out.

**Comment by ndoelger** [comment]
I'd love to talk about working on this with you! If you want you can shoot me an email at doelgern@gmail.com

**Reply by johnhuichen** [reply]
That's awesome! @ndoelger 

I will send you an email a bit later today :+1:

**Reply by aniketk33** [reply]
I'd also be interested in participating. You can reach me at aniketkumar693@gmail.com.

**Reply by mujgule-devika** [reply]
would love help out :) [mujgule.devika07@gmail.com ](mujgule.devika07@gmail.com )

**Reply by cold4Fun** [reply]
I would love to help as well. weiting4work@gmail.com

**Reply by ganand2021** [reply]
I would love to contribute as well! gopikoffc2020@gmail.com

**Reply by ngan-110** [reply]
I'd love to contribute as well :) ginnygreen110@gmail.com

**Reply by priyankthakur3** [reply]
I would like to contribute as well! priyankthakur3@live.com

**Reply by prnvjn** [reply]
Me too! - prnvjn@gmail.com

**Reply by Zfreeman24** [reply]
I'd like to contribute as well. [zacharyjames888@gmail.com](zacharyjames888@gmail.com)

**Reply by MayankTamakuwala** [reply]
I would like to contribute as well! [maytamaku.saidhwar@gmail.com](mailto:maytamaku.saidhwar@gmail.com)

**Comment by mujgule-devika** [comment]
Heyyy great idea, there's something similar out there already. Check this - https://jobpulse.fyi/?utm_source=linkedin)i/?utm_source=linkedin
Also would love to help out too :)

**Reply by johnhuichen** [reply]
They are actually doing exactly the same thing. Interesting, thanks @mujgule-devika

**Comment by umarani14** [comment]
I would love to contribute. ***@***.***

**Comment by iamsid22** [comment]
I’d love to contribute as well.
My email is: ***@***.***

Thank you.

**Comment by suuuuu017** [comment]
I would like to help! And I can be reached at ***@***.***
________________________________

**Comment by johnhuichen** [comment]
It's really amazing to see so many people wanting to help! It's definitely a big encouragement, and thank you for that!

The technical part of building a website is not a problem at all. I even know how long it's going to take to build the first iteration given a list of feature requirements. However at the moment I am checking if this website would even be useful once we built it. Nothing is worse than putting in months of work and the web traffic is flat.

I noticed that this repository links to a discord channel owned by cscareers.dev. As far as I can tell they already have an amazing website dedicated to the same cause (https://www.cscareers.dev/) so please check them out.

**Reply by mujgule-devika** [reply]
this makes a lot of sense. Also thanks for the resource, def looks very useful 😄
